lib/server/server.js

```diff
@@ -542,6 +547,28 @@ function execute(port) {
   });
 
   app.listen(port);
+
+  if (process.env.NODE_ENV !== 'production') {
+    const server = tinylr();
+    server.listen(LIVE_RELOAD_PORT, function() {
+      console.log('Livereload listening on %s...', LIVE_RELOAD_PORT);
+    });
+
+    // Gaze watches some specified files and triggers a callback when they change.
+    gaze(
+      ['../docs/**/*', 'blog/**/*', 'pages/**/*', 'static/**/*'],
```

Comment:
I think it would be good to listen on the entire `website` dir.

Reply:
👍